vita3k/gui/src/pkg_install_dialog.cpp

```diff
@@ -48,9 +48,24 @@ void draw_pkg_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
     if (draw_file_dialog) {
         result = host::dialog::filesystem::open_file(pkg_path, { { "PlayStation Store Downloaded Package", { "pkg" } } });
         draw_file_dialog = false;
-        if (result == host::dialog::filesystem::Result::SUCCESS)
-            ImGui::OpenPopup("install");
-        else if (result == host::dialog::filesystem::Result::CANCEL) {
+        if (result == host::dialog::filesystem::Result::SUCCESS) {
+            fs::ifstream infile(pkg_path.native(), std::ios::binary);
+            PkgHeader pkg_header{};
+            infile.read(reinterpret_cast<char *>(&pkg_header), sizeof(PkgHeader));
+            infile.seekg(sizeof(PkgHeader));
```

Comment:
I can't find further use of `infile` and I'm not sure if this line is needed

Reply:
Indeed.
Removed.